docs/devel/api_changes.md

```diff
@@ -485,6 +485,17 @@ To regenerate them:
 hack/update-generated-deep-copies.sh
 ```
 
+Since our components still use internal apis, you may need to generate code
```

Comment:
BTW - we are also getting rid of update-generated-conversions and incorporating it into update-codegen (this should be finished within a week), so maybe it makes sense to wait for that and update the whole doc.